package.json

```diff
@@ -113,6 +113,7 @@
     "terra-status-view": "^4.10.0",
     "terra-theme-context": "^1.0.0",
     "terra-theme-provider": "^4.0.0",
+    "terra-button": "^3.39.0",
```

Comment:
should we be pulling this in? i thought terra-button was already a dependency of terra-status-view

Reply:
Yes, terra-status-view does have terra-button as a dependency.
The only reason for adding terra-button as dependency here is due to the proptype definition [here](https://github.com/cerner/terra-application/pull/47/files#diff-cd1f5ff4f736d3f9cc1b643e78fe160eR14)